fastify.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const VERSION = '3.29.0'
+const VERSION = '3.29.1'
```

Comment:
Can you explain this change? If I'm not mistaken, we have some sort of build process that will automatically bump this (believe me, I don't like it).

Reply:
I don't think I actually bumped it last time, so the tests are failing now.